pom.xml

```diff
@@ -3,9 +3,9 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <!-- TODO switch to 1.532.2 if JENKINS-16936 is backported -->
-        <version>1.537</version>
+        <version>2.17</version>
     </parent>
+
```

Comment:
+1 please try to stage only functionnally impacted lines (use `git gui` or equivalent then select lines, then `Index lines`.